src/modules/deposit/entry-point/http/controller.ts

```diff
@@ -37,12 +48,4 @@ export class DepositController {
   getDepositsStats() {
     return this.depositService.getCachedGeneralStats();
   }
-
-  @Get("deposits/pending")
-  @ApiTags("deposits")
-  getPendingDeposits(@Query() query: GetDepositsQuery) {
-    const limit = parseInt(query.limit ?? "10");
-    const offset = parseInt(query.offset ?? "0");
-    return this.depositService.getPendingDeposits(limit, offset);
-  }
```

Comment:
@amateima I recall that this endpoint was intended to be used on the FE. With the new `/deposits/tx-page` endpoint, I assume that this not needed anymore